### PR TITLE
Update v10.31.0 changelog

### PR DESCRIPTION
**Pull request by tmashuang**
For #19032.

Should I have included all the MMI changes/additions?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,178 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- [MMI] interactive replacement token notification ([#18620](https://github.com/MetaMask/metamask-extension/pull/18620))
-- Update tokens controller ([#18459](https://github.com/MetaMask/metamask-extension/pull/18459))
-- Fixing issue with gasLimit in transaction being 0 ([#18682](https://github.com/MetaMask/metamask-extension/pull/18682))
-- 17873 ledger instruction banner ([#17937](https://github.com/MetaMask/metamask-extension/pull/17937))
-- Show Bridge button in TokenOverview component ([#18630](https://github.com/MetaMask/metamask-extension/pull/18630))
-- deps/security: vm2@3.9.16->3.9.17 ([#18707](https://github.com/MetaMask/metamask-extension/pull/18707))
-- Confirm recovery phrase onboarding unit test ([#18250](https://github.com/MetaMask/metamask-extension/pull/18250))
+
+### Fixed
+- Fix popover hiding ([#18843](https://github.com/MetaMask/metamask-extension/pull/18843))
```
Comment (danjm) [inline CHANGELOG.md:30]:
```suggestion
- Fix terms of use popover scroll button hiding when scollbar is at the bottom. ([#18843](https://github.com/MetaMask/metamask-extension/pull/18843))
```

**Review by danjm — CHANGES_REQUESTED** [review]
Only a couple suggested changes

File: CHANGELOG.md (hunk 1 above)
Comment (tmashuang) [reply CHANGELOG.md]: Done on https://github.com/MetaMask/metamask-extension/pull/19147/commits/8aa975ef9d3a67fd2c46570bccc9a79a322352d3#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR31

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,178 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Part of #17670 for file custom-spending-cap-tooltip.js ([#18686](https://github.com/MetaMask/metamask-extension/pull/18686))
-- Part of #17670 for file network-statistics.js ([#18644](https://github.com/MetaMask/metamask-extension/pull/18644))
-- Update slippage-buttons.js ([#18745](https://github.com/MetaMask/metamask-extension/pull/18745))
-- Update fee-card.js ([#18746](https://github.com/MetaMask/metamask-extension/pull/18746))
-- Update settings-tab.component.js ([#18753](https://github.com/MetaMask/metamask-extension/pull/18753))
-- Worked on #17670 & #18714 for 7 files. ([#18789](https://github.com/MetaMask/metamask-extension/pull/18789))
+- Remove Goerli buy link and disable button ([#18137](https://github.com/MetaMask/metamask-extension/pull/18137))
+- [FLASK] Don't show the title on Install/Update when it's loading ([#19012](https://github.com/MetaMask/metamask-extension/pull/19012))
+- [FLASK] Expand Snap authorship component ([#18775](https://github.com/MetaMask/metamask-extension/pull/18775))
 - [FLASK] Update permissions to use IconName and IconSize design system enum ([#18786](https://github.com/MetaMask/metamask-extension/pull/18786))
```
Comment (FrederikBolding) [inline CHANGELOG.md:27]:
```suggestion
```
Probably fine to omit.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,178 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Update keystone links ([#18792](https://github.com/MetaMask/metamask-extension/pull/18792))
-- Part of #17670 for file custom-spending-cap-tooltip.js ([#18686](https://github.com/MetaMask/metamask-extension/pull/18686))
-- Part of #17670 for file network-statistics.js ([#18644](https://github.com/MetaMask/metamask-extension/pull/18644))
-- Update slippage-buttons.js ([#18745](https://github.com/MetaMask/metamask-extension/pull/18745))
-- Update fee-card.js ([#18746](https://github.com/MetaMask/metamask-extension/pull/18746))
-- Update settings-tab.component.js ([#18753](https://github.com/MetaMask/metamask-extension/pull/18753))
-- Worked on #17670 & #18714 for 7 files. ([#18789](https://github.com/MetaMask/metamask-extension/pull/18789))
+- Remove Goerli buy link and disable button ([#18137](https://github.com/MetaMask/metamask-extension/pull/18137))
+- [FLASK] Don't show the title on Install/Update when it's loading ([#19012](https://github.com/MetaMask/metamask-extension/pull/19012))
+- [FLASK] Expand Snap authorship component ([#18775](https://github.com/MetaMask/metamask-extension/pull/18775))
```
Comment (FrederikBolding) [inline CHANGELOG.md:26]:
```suggestion
```
Remove to be combined with another entry

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,178 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Only build LavaMoat runtime once ([#18768](https://github.com/MetaMask/metamask-extension/pull/18768))
-- Some linting issues in the file custom spending cap tooltip.js ([#18687](https://github.com/MetaMask/metamask-extension/pull/18687))
-- Fix ICON_NAMES error for snaps settings ([#18784](https://github.com/MetaMask/metamask-extension/pull/18784))
-- Prevent new JS files in shared folder ([#17737](https://github.com/MetaMask/metamask-extension/pull/17737))
-- update icons to ts enum version ([#18698](https://github.com/MetaMask/metamask-extension/pull/18698))
-- Shallow git clone ([#18491](https://github.com/MetaMask/metamask-extension/pull/18491))
-- deps: bump serve-handler@6.1.3->6.1.5; minimatch@3.0.4->3.1.2 ([#18734](https://github.com/MetaMask/metamask-extension/pull/18734))
-- Bump `@metamask/slip44` to `3.0.0` ([#18773](https://github.com/MetaMask/metamask-extension/pull/18773))
-- [FLASK] Fix text selection bug in snap ui ([#18719](https://github.com/MetaMask/metamask-extension/pull/18719))
 - [FLASK] Add updated version of the Snaps settings UI ([#18438](https://github.com/MetaMask/metamask-extension/pull/18438))
```
Comment (FrederikBolding) [inline CHANGELOG.md:28]:
```suggestion
- [FLASK] Add updated version of the Snaps settings UI ([#18438](https://github.com/MetaMask/metamask-extension/pull/18438), ([#18775](https://github.com/MetaMask/metamask-extension/pull/18775)))
```
Combine with another PR

**Pull request merged**